projects/mac_ai/testing/ramalama.py

```
@@ -44,26 +52,65 @@
+def download_ramalama(base_work_dir, version, git_ref):
+    repo_url = config.project.get_config("prepare.ramalama.repo.url")
+
+    if version:
+        source = "/".join([
+            repo_url,
+            "archive/refs/tags",
+            f"{version}.zip",
+        ])
+        run.run_toolbox(
+            "remote", "download",
+            source=source, dest=dest,
```

Ruff (0.11.9)

Undefined name `dest`

(F821)